swagger.json

```diff
@@ -1121,6 +1121,122 @@
             }
           }
         },
+        "/fleet/drivers/{driver_id}/hos_certified_day_chart/{time_ms}": {
```

Comment:
Nit: could you make `chart` plural? (collections of resources should be plural form)